aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -2687,7 +2687,7 @@ impl AptosSimulationVM {
     pub fn create_vm_and_simulate_signed_transaction(
         transaction: &SignedTransaction,
         state_view: &impl StateView,
-    ) -> (VMStatus, TransactionOutput) {
+    ) -> (VMStatus, TransactionOutput, AptosSimulationVM) {
```

Comment:
Why do we return the VM? If we need it in the caller, please
1. Have a method to create simulation VM,
2. Have a method to simulate signed transaction

Reply:
(Not strictly needed because the code to inject abort info can be just moved inside)